artifact-app/build.rs

```diff
@@ -53,8 +55,21 @@ fn check_deps() {
 
 fn build_mdbook() {
     println!("Building the book");
+
+    let mdbook_dir = if cfg!(windows) {
+
+        let regex = Regex::new("[a-zA-Z].+").unwrap();
```

Comment:
nit: I think i would like this _slightly_ better if it were `r"[a-zA-Z]+:.*"` because at least I know I'm looking at a windows drive (`C:/`). However, it's up to you.